pom.xml

```diff
@@ -233,9 +255,9 @@
                 <version>${javax-activation.version}</version>
             </dependency>
             <dependency>
-                <groupId>log4j</groupId>
-                <artifactId>log4j</artifactId>
-                <version>${log4j.version}</version>
+                <groupId>ch.qos.reload4j</groupId>
+                <artifactId>reload4j</artifactId>
```

Comment:
Sounds silly, but as there are no tests that check the logging works as expected, have you run locally and verified the logs are still being written?

Reply:
Just a thought that maybe we should run a manual before / after and try to verify the outputs are the same or near enough, because we don't have automated logging testing

Reply:
I've run through the tests with this PR and `develop` and compared the output. There's no differences which are related to the logging change.